setuptools/command/develop.py

```diff
@@ -123,14 +127,43 @@ def install_for_development(self):
             self.easy_install(setuptools.bootstrap_install_from)
             setuptools.bootstrap_install_from = None
 
+        if self.distribution.namespace_packages and self.use_pth_file:
+            self._create_nspth()
+        else:
+            self._create_egg_link()
+
+        # postprocess the installed distro, fixing up .pth, installing scripts,
+        # and handling requirements
+        self.process_distribution(None, self.dist, not self.no_deps)
+
+    def _create_nspth(self):
+        namespace_packages = self._get_all_ns_packages()
+
+        # TODO Give the nspth it's own variable
+        filename, ext = os.path.splitext(self.egg_link)
+        filename += '-nspkg-develop.pth'
+        self.outputs.append(filename)
+
+        log.info("Installing %s (link to %s)", filename, self.egg_path)
+
+        # TODO: support package_dir.
+        lines = [
+            self._gen_nspkg_line(package) for package in namespace_packages]
+
+        if self.dry_run:
+            # always generate the lines, even in dry run
+            list(lines)
```

Comment:
Nothing. This was just left over from where I copied this code from `install_egg_info`. In that function it uses `map` to generate the lines instead of a list comprehension.